### PR TITLE
Remove obsolete callback

### LINKED ISSUE #66: OAS : /v1/regard-application : callback /v1/list-ltps-and-fcs could be removed
The service /v1/regard-application has two callbacks 
1. _NewApplicationCausesRequestForLatestTopologyInformation_ - which calls /v1/list-ltps-and-fcs of the regarded application
2. _NewApplicationCausesRequestForTopologyChangeInformation_ - which calls /v1/redirect-topology-change-information of the regarded application

Problem description:
When the regarded application receives /v1/redirect-topology-change-information, ALT receives the regarded application's data through /v1/update-all-ltps-and-fcs (a callback of /v1/redirect-topology-change-information) 
However, the request-body of /v1/update-all-ltps-and-fcs and the response-body of /v1/list-ltps-and-fcs are most likely having the same content which might lead ALT receiving the same data twice.

Possible solution:
The callback __NewApplicationCausesRequestForLatestTopologyInformation__ from /v1/regard-application can be removed in order to avoid redundancy.

### PR DESCRIPTION
**Pull request by openBackhaul**
NewApplicationCausesRequestForLatestTopologyInformation  removed from /v1/regard-application

Fixes #66

### CODE REVIEW

**Comment by openBackhaul** [comment]
Will create a conflict after merging pull-request "Reference CONFIGfile structure in callback uri" #130.
Deleting the affected callback is the correct resolution.

File: ApplicationLayerTopology+forwardings.yaml [hunk 1]
```diff
@@ -61,8 +61,6 @@ forwardings:
         uuid: alt-2-0-0-op-s-bm-005
       - server-name: /v1/update-client
         uuid: alt-2-0-0-op-s-bm-007
-      - server-name: /v1/list-ltps-and-fcs
```
Comment (IswaryaaS) [inline ApplicationLayerTopology+forwardings.yaml:64]: While removing entries for /v1/list-ltps-and-fcs, the corresponding entry of /v1/list-ltps-and-fcs :: alt-2-0-0-op-s-bm-008 in ServiceRequestCausesLoggingRequest is also deleted in forwardings file. It needs to be added again.

File: ApplicationLayerTopology.yaml [hunk 2]
```diff
@@ -3273,7 +3140,7 @@ paths:
...
+          url: #[/core-model-1-4:network-control-domain/control-construct=alt-2-0-0/logical-termination-point=alt-2-0-0-tcp-c-*/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-protocol]://[/core-model-1-4:network-control-domain/control-construct=alt-2-0-0/logical-termination-point=alt-2-0-0-tcp-c-*/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-address/domain-name or /core-model-1-4:control-construct/logical-termination-point=alt-2-0-0-tcp-c-*/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-address/ip-address/ipv-4-address]:[/core-model-1-4:network-control-domain/control-construct=alt-2-0-0/logical-termination-point=alt-2-0-0-tcp-c-*/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-port][/core-model-1-4:network-control-domain/control-construct=alt-2-0-0/logical-termination-point=alt-2-0-0-op-c-is-*-*-*-*-000/layer-protocol=0/operation-client-interface-1-0:operation-client-interface-pac/operation-client-interface-configuration/operation-name]
```
Comment (IswaryaaS) [inline ApplicationLayerTopology.yaml:3143]: As mentioned in comment https://github.com/openBackhaul/ApplicationLayerTopology/pull/130/#discussion_r1039180709, can we make the uuid(s) in URI to be specific to OKM application. Kindly let me know your views

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: ApplicationLayerTopology.yaml (hunk 2 above)
Comment (IswaryaaS) [reply ApplicationLayerTopology.yaml:3143]: This comment will become obsolete from decision made in https://github.com/openBackhaul/ApplicationLayerTopology/issues/59#issuecomment-1336986967. Hence resolving this conversation

File: ApplicationLayerTopology+forwardings.yaml (hunk 1 above)
Comment (openBackhaul) [reply ApplicationLayerTopology+forwardings.yaml:64]: Changed as proposed.
Many thanks for indicating.

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**